java/core/src/test/java/com/google/protobuf/CodedInputStreamTest.java

```diff
@@ -445,6 +443,82 @@ public void testReadMaliciouslyLargeBlob() throws Exception {
     }
   }
 
+  /**
+   * Test we can do messages that are up to CodedInputStream#DEFAULT_SIZE_LIMIT
+   * in size (2G or Integer#MAX_SIZE).
+   * @throws IOException
+   */
+  public void testParseMessagesCloseTo2G() throws IOException {
+    byte [] serializedMessage = getBigSerializedMessage();
```

Comment:
no space between "byte" and "[]". There are a few more occurrences in the code below.